2012-12-14 17:09:14
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-12-14 17:09:41
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-12-14 17:09:45
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-12-14 17:09:47
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-12-14 17:09:50
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-12-14 17:09:55
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-12-14 17:09:57
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-12-14 17:09:58
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-12-14 17:10:03
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-12-14 17:10:07
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-12-14 17:10:10
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-12-14 17:10:11
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
2012-12-14 17:10:15
Traceback (most recent call last):
  File "./sublimesvn/threads.py", line 18, in run_with_except_hook
  File "./sublimesvn/listeners.py", line 30, in run
  File "./sublimesvn/proc.py", line 71, in get_wc_root
  File "./sublimesvn/proc.py", line 172, in find_wc_root
ProcessError: svn: The path '.' appears to be part of a Subversion 1.7 or greater
working copy.  Please upgrade your Subversion client to use this
working copy.
